pubspec.lock

```diff
@@ -0,0 +1,528 @@
+# Generated by pub
```

Comment:
this file is new. Do we want it?

Reply:
just me lazily staging all files, I'll remove it